### PR TITLE
Add Kazuaki Ishizaki to the list of committers

### PR DESCRIPTION
**Pull request by kiszk**

### CODE REVIEW

**Comment by kiszk** [comment]
ping @gatorsmile

**Review by HyukjinKwon — APPROVED** [review]
Congrats!

**Comment by HyukjinKwon** [comment]
@kiszk, would you like to push it by yourself or should I?

**Comment by kiszk** [comment]
~@HyukjinKwon thanks, could you please merge it?~ Let me do it myself for my exercise.

File: committers.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ navigation:
 |Yin Huai|Databricks|
 |Shane Huang|Intel|
 |Dongjoon Hyun|Hortonworks|
+|Kazuaki Ishizaki|IBM|
```
Comment (gatorsmile) [inline committers.md:37]: Congratulations!

Comment (kiszk) [reply committers.md:37]: Thank you!

**Comment by dongjoon-hyun** [comment]
Congratulation, @kiszk ! :D

**Comment by kiszk** [comment]
@dongjoon-hyun thank you!

**Pull request closed without merging**